packages/pwa-buildpack/envVarDefinitions.json

```diff
@@ -11,6 +11,18 @@
                 }
             ]
         },
+        {
+            "name": "Magento Store Edition",
+            "variables": [
+                {
+                    "name": "MAGENTO_BACKEND_EDITION",
```

Comment:
What if we could do this automatically through schema validation instead of having to use a flag?

I have a PR that gets the validation to function -- and it does alert if a fragment/component has invalid gql.

Reply:
If the schema validation happens before bundling, we can definitely use that.